### PR TITLE
Add netlify redirect for `/documentation`

### PR DESCRIPTION
**Pull request by patocallaghan**
Fixes https://github.com/ember-learn/ember-website/issues/147

* Adds netlify redirect
* Removes `documentation` specific code
* Removes `documentation` entry from router.js
* Removes `documentation` prember entry

Gif of redirect working, e.g. https://deploy-preview-166--ember-beta-website.netlify.com/documentation/object-model/classes-and-instances/
docs-redirect

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,8 @@
 /testem.log
 /yarn-error.log
 .DS_Store
+.vscode
```
Comment (patocallaghan) [inline .gitignore]: This was annoying me but I'd be happy to pull this out to a separate PR if you don't think it fits here?

Comment (mansona) [reply .gitignore]: no that's completely fine 🎉
